packages/rancher-cis-benchmark/charts/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
-appVersion: v0.0.1
+appVersion: v0.2.0
 description: The cis-operator enables running CIS benchmark security scans on a kubernetes
   cluster
 name: rancher-cis-benchmark
-version: 0.0.1
+version: 0.2.0
```

Comment:
Are we removing old version of chart ?

Reply:
yes we want to start the app version start from v0.2.0 - I hope we need to change both version and appVersion fields?

Reply:
The version corresponds to the version of the chart, the appVersion is the version of the operator. Thats why we often use the double-zero method and keep them in line. IE: operator version 1.2.1 gets `appVersion` of 1.2.1 and `version` of 1.2.100.

If we are removing the old version of the chart, you'll need to remove from the dev-v2.5 branch the entry in `assets/index.yaml`, and the assets.

Reply:
Ok, Added this new PR for assets changes for dev-v2.5

https://github.com/rancher/charts/pull/716